doc/general_documentation/readme.md

```diff
@@ -0,0 +1,98 @@
+\page doc Documentation
+
+# Tools used for FSW documentation
+
+## Code documentation
+
+Code should be documented using doxygen.
+
+*Add here an explanation of the overall structure and how to create subsystem
```

Comment:
TODO

Reply:
I did not write that and I have no idea what the person had in mind with it, removed it. If someone is doing documentation, then I assume they know the fsw relatively well and can look at other packages for examples.